external_notifications_log.tf

```diff
@@ -0,0 +1,53 @@
+locals {
+  external_notifications_queue_name     = "${local.environment}-external-notifications"
```

Comment:
Since these have essentially the same value, could there be one main variable that these 3 make reference to?

Reply:
Yeah ok. I've changed it to `external_notifications_name`